packages/story-editor/src/app/helpCenter/useHelpCenter/effects.js

```diff
@@ -77,25 +76,26 @@ export const createDynamicNavigationFlow = (previous, next) => {
   if (!isComingFromMenu(previous, next)) {
     return {};
   }
-  const appendedTips = BASE_NAVIGATION_FLOW
+  const appendedTips = next.navigationFlow
```

Comment:
I think the expected behavior could be documented a little better here, sorry.

So one of the UX features we have to accommodate here is appending all the unread tips to the end so a user can click into any tip, then press `next` all the way until they get to the last tip and have viewed all unread tips.

ie:
- lets say tips are [tip1, tip2, tip3, tip4] and they're all unread
- user opens the main menu
- user clicks into tip3 first
- the resulting navigation flow should look like: [tip1, tip2, tip3, tip4, tip1, tip2]

anyway, I think you can just update this to be the following and it should be fine:
 ```
   const baseNavigationFlow = next.tipKeys;
   const appendedTips = baseNavigationFlow;
    // Tips before current index
    .slice(0, next.navigationIndex)
    // Filter out read tips
    .filter((tip) => !next.readTips[tip]);
  return {
    navigationFlow: [
      ...baseNavigationFlow,
      ...appendedTips,
      DONE_TIP_ENTRY[0],
    ],
  };
 ```

Reply:
you can test this behavior out manually by clearing all your read tips as a user by typing this into the console to clear all your read tips on the backend:
```
await wp.apiFetch({ path: '/web-stories/v1/users/me', method: 'POST', data: { meta: { web_stories_onboarding: {} }}})
```

Reply:
Thanks updated